src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -11,8 +11,9 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be 8 digits long "
+                    + "according to Singapore mobile number format";
+    public static final String VALIDATION_REGEX = "\\d{8}";
```

Comment:
This says that it should be exactly 8 digits (previously it was exactly 3 or more digits), is that alright?

Reply:
yeap because Singapore phone number is 8 digits

Reply:
We are only considering Singapore phone numbers since we based this off from NUS (Singapore)